.env.example

```
@@ -27,3 +27,7 @@ ORACLE_STATSD_ADDR="localhost:8125"
 ORACLE_STATSD_STUCK_DUR="5m"
 ORACLE_STATSD_MOCKING=false
 ORACLE_STATSD_DISABLED=false
+
+STORK_WEBSOCKET_URL="wss://dev.api.stork-oracle.network/evm/subscribe"
+STORK_WEBSOCKET_HEADER=
+STORK_WEBSOCKET_SUBSCRIBE_MESSAGE={"type":"subscribe","trace_id":"123","data":["%s"]}"
```

I would send  uuid here, feels weird using always the same trace id
```suggestion
STORK_WEBSOCKET_SUBSCRIBE_MESSAGE={"type":"subscribe","trace_id":"%s","data":["%s"]}"
```